src/traits.c

```diff
@@ -385,7 +385,7 @@ Expression *pointerBitmap(TraitsExp *e)
             d_uns64 arrayoff = offset;
             d_uns64 nextsize = t->next->size();
             d_uns64 dim = t->dim->toInteger();
-            for (size_t i = 0; i < dim; i++)
+            for (size_t i = 0; i < (size_t)dim; i++)
```

Comment:
I don't really agree but it's not worth arguing about.  I've updated the pull with the code you suggested.